dbcon/joblist/tupleaggregatestep.cpp

```diff
@@ -1537,7 +1534,7 @@ void TupleAggregateStep::prep1PhaseAggregate(
     }
 
     // there is avg(k), but no count(k) in the select list
-    uint64_t lastCol = returnedColVec.size();
+    uint64_t lastCol = outIdx;
```

Comment:
This fixes the bug exists since MCOL-1201 and fires back for ExeMgr-side UDAF processing only.
I didn't put it into a separate issue b/c nobody complains since that time.